docs/containers/quickstart-aspnet-core.md

```diff
@@ -58,12 +58,12 @@ In this guide you will learn how to:
      Determining projects to restore...
      All projects are up-to-date for restore.
      net -> C:\source\repos\net\bin\Debug\net7.0\net.dll
-   
+
    Build succeeded.
        0 Warning(s)
        0 Error(s)
-   
```

Comment:
The editor *really* wanted to do these couple lines of whitespace changes, I literally couldn't stop it from doing them when editing in the browser...may as well just let it through